Code/Mantid/Framework/PythonInterface/test/python/plugins/algorithms/DNSDetEffCorrVanaTest.py

```diff
@@ -0,0 +1,130 @@
+import unittest
+from testhelpers import run_algorithm
+from mantid.api import AnalysisDataService
+import numpy as np
+import mantid.simpleapi as api
+from mantid.simpleapi import DNSDetEffCorrVana
+
+
+class DNSDetEffCorrVanaTest(unittest.TestCase):
+    __dataws = None
+    __vanaws = None
+    __bkgrws = None
+
+    def _create_fake_workspace(self, wsname, dataY):
```

Comment:
If fake data creator methods like this one become useful for more tests at some point you can add test helpers under `Framework/PythonInterface/test/testhelpers/`. So you could have a MLZ / DNS data creation helper if that's convenient.